android/app/src/main/java/com/microsoft/codepush/common/managers/SettingsManager.java

```diff
@@ -69,17 +71,17 @@ public SettingsManager(Context applicationContext, CodePushUtils codePushUtils)
 
     /**
      * Gets an array with containing failed updates info arranged by time of the failure ascending.
-     * Each item represents an instance of {@link CodePushLocalPackage} that has failed to update.
+     * Each item represents an instance of {@link CodePushPackage} that has failed to update.
      *
      * @return an array of failed updates.
      */
-    public ArrayList<CodePushLocalPackage> getFailedUpdates() {
+    public ArrayList<CodePushPackage> getFailedUpdates() {
```

Comment:
Why do we use `CodePushPackage` here instead of `CodePushLocalPackage`?

Reply:
@annakocheshkova because originally acquisition sdk uses `CodePushPackage` as container for failed updates info.